backend/src/zac/conf/includes/base.py

```diff
@@ -418,6 +418,9 @@
         "rest_framework.authentication.SessionAuthentication",
         "rest_framework.authentication.TokenAuthentication",
     ),
+    "DEFAULT_PARSER_CLASSES": (
+        "djangorestframework_camel_case.parser.CamelCaseJSONParser",
+    ),
```

Comment:
```suggestion
    "DEFAULT_PARSER_CLASSES": (
        "djangorestframework_camel_case.parser.CamelCaseJSONParser",,
        "rest_framework.parsers.FormParser",
        "rest_framework.parsers.MultiPartParser",
    ),
```
you lost a couple of other parsers :grimacing: